pycbc/workflow/jobsetup.py

```diff
@@ -1225,7 +1488,7 @@ def create_node(self, bank):
             curr_tag = 'bank%d' %(i)
             # FIXME: What should the tags actually be? The job.tags values are
             #        currently ignored.
-            curr_tags = bank.tags + [curr_tag]
+            curr_tags = bank.tags + [curr_tag] + self.tags
```

Comment:
Why the need to add self.tags here? This differs from the approach with SplitInspinj (and one might want to combine those two codes/set up functions in the future). I would again suggest the approach from SplitInspinj also here.